include/zephyr/bluetooth/l2cap.h

```diff
@@ -194,7 +194,7 @@ struct bt_l2cap_le_chan {
 	struct bt_l2cap_le_endpoint	tx;
 #if defined(CONFIG_BT_L2CAP_DYNAMIC_CHANNEL)
 	/** Channel Transmission queue */
-	struct k_fifo                   tx_queue;
+	struct k_fifo                   sdu_tx_queue;
```

Comment:
this is not a breaking change. It's not public API in the first place.

Reply:
Why isn't it a public API? It's placed in `include/zephyr` where our public APIs are

Reply:
But the fact is that `tx_queue` was never marked as internal in any way, and we don't know if any implementations out there use it for whatever. 

I'm not going to fight this hard, as I agree that this probably shouldn't be considered a public API, and this sort of some-fields-are-public-and-some-are-private-but-wont-tell design that L2CAP (and GATT and ISO have as well to some degree) is dangerous because we may consider them private and users may consider them public, which is why I've also pushed for the opaque data type design (like `bt_conn`) where it's significantly more clear what is public and what is private. 

If I'm the only one who consider this a public API change, then let it be

Reply:
> which is why I've also pushed for the opaque data type design

Dudes from google get mad when I do this. I tried, the guy nack'd the PR and unsubscribed :/

Reply:
reverted the rename